scrooge-generator/src/main/scala/com/twitter/mustache/ScalaObjectHandler.scala

```diff
@@ -0,0 +1,69 @@
+package com.twitter.mustache
```

Comment:
This class was copied directly from Mustache.java because it wasn't released for Scala 2.11 on Java 7.